### ISSUE TITLE
Switch from node-sass to dart-sass

### ISSUE THREAD

**Issue by twschiller** [pixiebrix/pixiebrix-extension#975]
**Context**
- https://sass-lang.com/blog/libsass-is-deprecated
- dart-sass has support for modern features (e.g., `@use`), https://sass-lang.com/documentation/at-rules/use

**Comment by fregante** [comment]
Blocked by:

- https://github.com/twbs/bootstrap/issues/29219#issuecomment-772714040

**Comment by fregante** [comment]
I investigated this and yes, probably switching to Bootstrap 5 would fix these issues, but from what I see we can't do that because we vendored the entire theme. That's why when I tried to install v5 I still got errors.

Example: https://github.com/pixiebrix/pixiebrix-extension/blob/a5521ed24ced6bb1a731ad37a5947e32acb76b4d/src/vendors/theme/assets/styles/_variables.scss#L21-L37

Fixed in v5 a few years ago: https://github.com/twbs/bootstrap/pull/23701

---

In short, the switch is blocked by:

- Blocked by https://github.com/pixiebrix/pixiebrix-extension/issues/1072

**Comment by fregante** [comment]
Yes that's the one I used in https://github.com/pixiebrix/pixiebrix-app/pull/465 as well

**Comment by fregante** [comment]
If desired, I suppose I can fix the issues in the vendored theme. I don’t think we're going to "upgrade" the theme to a later version at this point.
